### PR TITLE
Timetable

### PR DESCRIPTION
**Pull request by DhanshreeK**
Timetable pull request

### CODE REVIEW

File: app/views/time_tables/new_timetable.html.erb [hunk 1]
```diff
@@ -25,13 +25,13 @@
...
               
-          <div class="col-md-7"> <%= f.date_select :start_date,:use_month_numbers=>true,:start_year => Date.current.year-1, :end_year => Date.current.year+1,:order=>[:day,:month,:year]%></div>
+          <div class="col-md-7"> <%= f.date_select :start_date,:use_month_numbers=>true,:start_year =>1.years.ago.year, :end_year => 1.years.since.year,:order=>[:day,:month,:year],:include_blank => false%></div>
           </div>
           <br>
        <div class="form-group">
          <div class="col-md-3"><b>End Date:</b></div>
            
-          <div class="col-md-7"> <%= f.date_select :end_date,:use_month_numbers=>true,:start_year => Date.current.year-1,:end_year => Date.current.year+1,:order=>[:day,:month,:year]%>
+          <div class="col-md-7"> <%= f.date_select :end_date,:use_month_numbers=>true,:start_year =>1.years.ago.year,:end_year =>1.years.since.year,:order=>[:day,:month,:year],:include_blank => false%>
```
Comment (swapnilv92) [inline app/views/time_tables/new_timetable.html.erb:34]: what is the purpose of `include_blank:` here?
Also use the latest syntax for ruby hash-
for ex.
`:include_blank => false` will change to `include_blank: false`

**Comment by swapnilv92** [comment]
- change the target branch to `datepucker_bugfix`
- squash commits into a single commit.

File: app/views/time_tables/new_timetable.html.erb (hunk 1 above)
Comment (DhanshreeK) [reply app/views/time_tables/new_timetable.html.erb:34]: Useful if there is not a default value required for the select element.

**Comment by DhanshreeK** [comment]
I have updated Timetable Branch again check the changes.

**Pull request closed without merging**